@here/harp-mapview/lib/MapView.ts

```diff
@@ -2042,6 +2106,40 @@ export class MapView extends THREE.EventDispatcher {
         }
     }
 
+    /**
+     * Set camera, so all target points (or [[GeoBox]]) are visible.
+     *
+     * If not specified current `tilt` and `heading` is used.
+     *
+     * Note.
+     *
+     * This methods uses best effort to find best view for given set of points without a priori
+     * knowledge of any relationship between points (like polygon ordering) or deciding which side
+     * of polygon shall be visible, so it doesn't support well inputs that have longitude span
+     * significantly bigger than 180.
+     *
+     * In particular, if you give set of points on both sides of antimeridian, with flat projection
+     * and `tileWrappingEnabled` feature it will use assume that map is wrapped around and will
+     * find "minimal" box in projected/wrapped world instead showing whole world.
+     *
+     * Options
+     *
+     *  - `tilt`   - optional, change tilt, defaults to current [[MapView]] tilt
+     *  - `heading` - optional, change heading, defaults to current [[MapView]] heading
+     *
+     * @param target [[GeoBox]] or list of points conforming to [[GeoCoordLike]]
+     */
+    fitBoundsXX(
```

Comment:
Why do we need this method again? Just convenience for users of other APIs?

Reply:
why XX?

Reply:
Leftover. I was sure i've finally removed it.